modules/reference/pages/feature/openidConnectServer/examples.adoc

```diff
@@ -1,16 +1,10 @@
 == Examples
 
-The OpenID Connect Provider (OP) feature can be configured to manage user authentication for OpenID Connect (OIDC) clients and microservices during development.
-The OP feature can work directly with a user registry, such as LDAP, or act as an identity broker and delegate to another security provider.
-In production, the application uses a cloud-hosted SSO provider.
-
-Any user that is authorized to use OIDC must also be mapped to the authenticated OAuth role.
-
 === Managing OAuth clients with `localStore`
 
 The following example shows how to create a basic configuration to run an OpenID Connect Provider.
 This configuration is especially useful during development and testing before deployment to the cloud, where the application typically uses a cloud-based SSO provider instead.
-The example uses a simple link:/docs/ref/feature/#appSecurity-3.0.html[basic registry], which is defined in the server configuration, to manage a single user:
+The example uses a simple xref:/feature/#appSecurity-3.0.adoc[basic registry], which is defined in the server configuration, to manage a single user:
```

Comment:
xref:/feature/#appSecurity-3.0.adoc[basic registry]

should be

xref:feature/appSecurity-3.0.adoc[basic registry]